geode-core/src/main/java/org/apache/geode/internal/lang/SystemUtils.java

```diff
@@ -50,6 +55,16 @@
 
   private static final String LINE_SEPARATOR = System.getProperty("line.separator");
 
+  /**
+   * Is the Java version the requested version.
+   *
+   * @param requiredVersion the required version
+   * @return true if the actual version is the required version
+   */
+  public static boolean isJavaVersion(final JavaVersion requiredVersion) {
```

Comment:
I'm leaning towards removing `isJavaVersion` and just have all callers use Apache Commons' JavaVersion directly.